doc/src/guides/solving/solve-diophantine-equation.md

```diff
@@ -0,0 +1,170 @@
+# Solve a Diophantine Equation Algebraically
+
+Use SymPy to solve a Diophantine equation (find integer solutions to a
+polynomial equation) algebraically, returning a parameterized general solution
+if possible. For example, solving the [Pythagorean
+theorem](https://en.wikipedia.org/wiki/Pythagorean_theorem) $a^2 + b^2 = c^2$
+yields $(a=2pq, b=p^2-q^2, c=p^2-q^2)$.
+
+## Alternatives to Consider
+
+There are few alternatives for finding a parameterized general solution a
+Diophantine equation. {func}`~.solve` simply solves for one variable in terms of
```

Comment:
I couldn't find any other open-source options for solving Diophantine equations with parametrized solutions. Does anyone else know of one?